sandbox/registration-processor-camel-routes-new-mz.xml

```diff
@@ -1,4 +1,66 @@
 <routes xmlns="http://camel.apache.org/schema/spring">
+	 <route id="packet-reciever-->securezone-notification new route">
+        <from uri="eventbus:packet-receiver-new-bus-out" />
+        <log
+            message="packet-reciever-->packet-uploader new ${bodyAs(String)}" />
+        <choice>
+            <when>
+                <simple>${bodyAs(String)} contains '"isValid":true'</simple>
+                <process ref="tokenGenerationProcessor"/>
+                <setHeader headerName="CamelHttpMethod">
+                    <constant>POST</constant>
+                </setHeader>
+                <setHeader headerName="Content-Type">
+                    <constant>application/json</constant>
+                </setHeader>
+                <setHeader headerName="Cookie">
+                    <simple>${header.Cookie}</simple>
+                </setHeader>
+                <setBody>
+                    <simple>${bodyAs(String)}</simple>
+                </setBody>
+                <to uri="http://mz.ingress:30080/registrationprocessor/v1/securezone/notification" />
+            </when>
+            <when>
+                <simple>${bodyAs(String)} contains '"internalError":true'</simple>
+                <to uri="eventbus:retry-bus-in" />
+            </when>
+            <otherwise>
+                <to uri="eventbus:error-bus-in" />
+            </otherwise>
+        </choice>
+    </route>	
+	 <route id="securezone notofication-->securezone notofication new route">
```

Comment:
Rename route id to "securezone-notification-->securezone-notification HTTP all process route"